pagopa/io-app
# chore: [IA-13] Changed all native base Input with LabelledItem generic component

## Short description
Changed all native base Input with LabelledItem generic component, this operation is necessary to manage input accessibility uniformly.

## How to test
Move into the screens that contains the new LabelledItem component and check if the previous behaviour has not changed and if the app works correctly.

As requested from @thisisjp, here are some screenshots examples showing the various input behaviours.
These screens only show some examples of input behaviour in general, it does not actually change the existing logic in this screen.

<img width="300" alt="Screenshot 2021-06-15 at 14 22 14" src="https://user-images.githubusercontent.com/61834253/122051973-77708600-cde5-11eb-9354-d17c576dfe3c.png">

<img width="300" alt="Screenshot 2021-06-15 at 14 25 31" src="https://user-images.githubusercontent.com/61834253/122052102-9c64f900-cde5-11eb-9bc8-2bd52b9b09ee.png">

<img width="300" alt="Screenshot 2021-06-15 at 14 26 20" src="https://user-images.githubusercontent.com/61834253/122052182-b30b5000-cde5-11eb-9e20-9fc1f9e3b491.png">

<img width="300" alt="Screenshot 2021-06-15 at 14 38 35" src="https://user-images.githubusercontent.com/61834253/122054581-18604080-cde8-11eb-86d4-806ec5ea7214.png">

<img width="300" alt="Screenshot 2021-06-15 at 14 37 07" src="https://user-images.githubusercontent.com/61834253/122053646-37aa9e00-cde7-11eb-85a6-0056b2358d65.png">

You can also add icon on the left or on the right:
<img width="300" alt="Screenshot 2021-06-15 at 14 33 01" src="https://user-images.githubusercontent.com/61834253/122053018-a509ff00-cde6-11eb-8ef1-297036698352.png">
<img width="300" alt="Screenshot 2021-06-15 at 14 33 55" src="https://user-images.githubusercontent.com/61834253/122053123-c0750a00-cde6-11eb-8e99-1bb83b30cf99.png">
